docs/build/_partials/_oracles_contract_data.mdx

```diff
@@ -28,8 +28,8 @@ import TabItem from '@theme/TabItem';
 
 You can find detailed documentation on the Oracles in their official documentation:
 
-* [Pyth Oracle Docs](https://docs.pyth.network/price-feeds/contract-addresses/evm) 
-* [Supra Pull Docs](https://gb-docs.supraoracles.com/docs/data-feeds/pull-model/networks)
-* [Supra Push Docs](https://gb-docs.supraoracles.com/docs/data-feeds/decentralized/networks)
+* [Pyth Oracle Docs](https://docs.pyth.network/price-feeds/contract-addresses/evm)
+* [Supra Pull Docs](https://docs.supra.com/docs/data-feeds/pull-modelv1)
```

Comment:
Just out of interest, I have no clue about all of this. But why not linke v2? Don't we support it? I guess using the latest version is always better? :trollface:

Reply:
Changing to V2... thanks